### PR TITLE
Added build script for ruamel-ordereddict

### PR DESCRIPTION
**Pull request by atharv-phadnis**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?

Package Name: **ruamel-ordereddict**
Package Version: **0.4.15**
Package URL: [ruamel-ordereddict](http://hg.code.sf.net/p/ruamel-ordereddict/code)

Tested on: **UBI 8.5**
Build Errors: **None**
Test Errors: **None**

### CODE REVIEW

**Comment by atharv-phadnis** [comment]
@Siddhesh-Ghadi please review

File: r/ruamel-ordereddict/ruamel-ordereddict_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=ruamel-ordereddict
+PACKAGE_VERSION=${1:-0.4.15}
+PACKAGE_URL=http://hg.code.sf.net/p/ruamel-ordereddict/code
+
+set -e
+
+yum -y update && yum install -y python2 python2-devel gcc gcc-c++ make redhat-rpm-config
+python2 -m pip install Mercurial
```
Comment (Siddhesh-Ghadi) [inline r/ruamel-ordereddict/ruamel-ordereddict_ubi_8.5.sh:27]: Is python2 hard requirement? Can we use python3?

Comment (atharv-phadnis) [reply r/ruamel-ordereddict/ruamel-ordereddict_ubi_8.5.sh:27]: @Siddhesh-Ghadi Yes, the package doesn't support python3, we can refer to comments on its issues like: https://sourceforge.net/p/ruamel-ordereddict/tickets/8/#ae3d

Comment (Siddhesh-Ghadi) [reply r/ruamel-ordereddict/ruamel-ordereddict_ubi_8.5.sh:27]: Okay. Thanks.

**Review by Siddhesh-Ghadi — APPROVED** [review]
LGTM

**Pull request merged**